### ISSUE TITLE
Indents on secondary lines are lost

### ISSUE THREAD

**Issue by Borvik** [helixquar/asciidecorator#1]
If the text selected to change is indented (say in a comment indented to align with a function in a class), the first line of the generated ASCII font is indented as expected, but every following line has no indent and must be indented manually.

```javascript
describe('Tests', function() {
  /*
  Test
  */
  describe('Express', function() {
  });
});
```
becomes
```javascript
describe('Tests', function() {
  /*
  888888 888888 .dP"Y8 888888 
  88   88__   `Ybo."   88   
  88   88""   o.`Y8b   88   
  88   888888 8bodP'   88   
  */
  describe('Express', function() {
  });
});
```
instead of 
```javascript
describe('Tests', function() {
  /*
  888888 888888 .dP"Y8 888888 
    88   88__   `Ybo."   88   
    88   88""   o.`Y8b   88   
    88   888888 8bodP'   88   
  */
  describe('Express', function() {
  });
});
```

**Comment by mistasse** [comment]
#3 would solve it, right?

**Comment by Borvik** [comment]
I would tend to thinks so.

**Comment by helixquar** [comment]
Fixed by #3 .
